ts/features/bonus/cgn/components/detail/eyca/EycaErrorComponent.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import { View } from "native-base";
+import { StyleSheet } from "react-native";
+import { H4 } from "../../../../../../components/core/typography/H4";
+import I18n from "../../../../../../i18n";
+import IconFont from "../../../../../../components/ui/IconFont";
+import { IOColors } from "../../../../../../components/core/variables/IOColors";
+import ButtonDefaultOpacity from "../../../../../../components/ButtonDefaultOpacity";
+import { Label } from "../../../../../../components/core/typography/Label";
+import { InfoBox } from "../../../../../../components/box/InfoBox";
+
+type Props = {
+  onRetry: () => void;
+};
+
+const styles = StyleSheet.create({
+  rowBlock: {
+    flexDirection: "row"
+  },
+  spaced: {
+    justifyContent: "space-between"
+  }
+});
+
+const TITLE_ICON_SIZE = 24;
+const INFOBOX_ICON_SIZE = 32;
+
+const EycaErrorComponent: React.FunctionComponent<Props> = (props: Props) => (
```

Comment:
Could leave a description of this component?

Reply:
Fixed in 059777b